src/multitaper.jl

```diff
@@ -576,26 +575,27 @@ end
     else
         mt_fft_tapered_multichannel!(x_mt, signal, config)
     end
-    x_mt[1, :, :] ./= sqrt(2)
+    @views x_mt[1, :, :] ./= sqrt(2)
```

Comment:
```suggestion
    x_mt[1, :, :] ./= sqrt(2)
```
The broadcast uses a `dotview` anyway, and the extra `@views` seems to make things worse, actually. (Apart from the that, this is one of those unrelated changes I'm not exactly happy about.)

Reply:
You're right. Don't know what I benchmarked yesterday, but indeed the `@views` version is faster and doesn't allocate. Sorry for the noise.